src/com/lilithsthrone/game/Game.java

```diff
@@ -518,22 +513,17 @@ public static void exportGame(String exportFileName, boolean allowOverwrite) {
 		dir.mkdir();
 		
 		boolean overwrite = false;
-		if (dir.isDirectory()) {
-			File[] directoryListing = dir.listFiles((path, filename) -> filename.endsWith(".xml"));
-			if (directoryListing != null) {
-				for (File child : directoryListing) {
-					if (child.getName().equals(exportFileName+".xml")){
-						if(!allowOverwrite) {
-							Main.game.flashMessage(PresetColour.GENERIC_BAD, "Name already exists!");
-							return;
-						} else {
-							overwrite = true;
-						}
-					}
-				}
+		dir = new File("data/saves/"+exportSaveName);
+		if(dir.exists()) {
+			if(!allowOverwrite) {
+				Main.game.flashMessage(PresetColour.GENERIC_BAD, "Save already exists!");
+				return;
+			} else {
+				overwrite = true;
 			}
 		}
-
+		dir.mkdir();
```

Comment:
Looks like this slipped through when I was experimenting with per character folders

Reply:
This probably stems from the merge and needs to be looked into.